src/pkgcore/ebuild/ebuild_src.py

```diff
@@ -314,7 +314,8 @@ def mandatory_phases(self):
 
     @property
     def live(self):
-        return bool(const.VCS_ECLASSES.intersection(self.inherited))
+        return ('live' in self.properties or
+                bool(const.VCS_ECLASSES.intersection(self.inherited)))
```

Comment:
Why keep the eclasses here?

Reply:
In part because pkgcheck was using it as well for tests.